win32ss/user/ntuser/scrollbar.c

```diff
@@ -1334,60 +1337,57 @@ NtUserSetScrollInfo(
    PWND Window = NULL;
    NTSTATUS Status;
    SCROLLINFO ScrollInfo;
-   DECLARE_RETURN(DWORD);
+   DWORD Ret = 0;
    USER_REFERENCE_ENTRY Ref;
 
    TRACE("Enter NtUserSetScrollInfo\n");
    UserEnterExclusive();
 
    if(!(Window = UserGetWindowObject(hWnd)) || UserIsDesktopWindow(Window) || UserIsMessageWindow(Window))
-      RETURN(0);
+      goto Cleanup;
 
    UserRefObjectCo(Window, &Ref);
 
    Status = MmCopyFromCaller(&ScrollInfo, lpsi, sizeof(SCROLLINFO) - sizeof(ScrollInfo.nTrackPos));
    if(!NT_SUCCESS(Status))
    {
       SetLastNtError(Status);
-      RETURN(0);
+      goto Cleanup;
    }
 
-   RETURN(co_IntSetScrollInfo(Window, fnBar, &ScrollInfo, bRedraw));
+   Ret = co_IntSetScrollInfo(Window, fnBar, &ScrollInfo, bRedraw);
 
-CLEANUP:
+Cleanup:
    if (Window)
       UserDerefObjectCo(Window);
 
-   TRACE("Leave NtUserSetScrollInfo, ret=%lu\n", _ret_);
+   TRACE("Leave NtUserSetScrollInfo, ret=%lu\n", Ret);
    UserLeave();
-   END_CLEANUP;
+   return Ret;
 }
 
 DWORD APIENTRY
 NtUserShowScrollBar(HWND hWnd, int nBar, DWORD bShow)
 {
    PWND Window;
-   DECLARE_RETURN(DWORD);
-   DWORD ret;
+   DWORD ret = 0;
    USER_REFERENCE_ENTRY Ref;
 
    TRACE("Enter NtUserShowScrollBar\n");
    UserEnterExclusive();
 
-   if (!(Window = UserGetWindowObject(hWnd)))
-      RETURN(0);
-
-   UserRefObjectCo(Window, &Ref);
-   ret = co_UserShowScrollBar(Window, nBar, (nBar == SB_VERT) ? 0 : bShow,
+   Window = UserGetWindowObject(hWnd);
+   if (Window)
+   {
+      UserRefObjectCo(Window, &Ref);
+      ret = co_UserShowScrollBar(Window, nBar, (nBar == SB_VERT) ? 0 : bShow,
                                             (nBar == SB_HORZ) ? 0 : bShow);
```

Comment:
Needs also its indent modification (to keep up with original code):
```suggestion
                                               (nBar == SB_HORZ) ? 0 : bShow);
```